routes/views/director_profile.js

```diff
@@ -12,22 +12,27 @@ exports = module.exports = (req, res) => {
     locals.director = {};
     locals.movies = [];
 
-    Director.model.findOne({ key: req.params.name, }).exec((err, director) => {
-
-      if (err || !director) {
-        res.locals.title = '404 error | YiMovi';
-        res.status(404).render('errors/404');
-        return;
-      }
-
-      locals.director = director;
-
-      getMoviesBy('director', director, (moviesErr, movies) => {
-        if (moviesErr) return next(moviesErr);
-        locals.movies = locals.movies.concat(movies || []);
-        next();
+    Director.model
+      .findOne({ key: req.params.name, })
+      .exec((err, director) => {
+
+        if (err || !director) {
+          res.locals.title = '404 error | YiMovi';
+          res.status(404).render('errors/404');
+          return;
+        }
+
+        locals.director = require('../helpers/localize_results.js')
+        .localizeResults(locals.locale, director);
+
+        locals.title = `${director.name} ${director.name_chn || ''} | YiMovi director profile`;
```

Comment:
This title key should be removed, the view builds the title name + static fields from JSON files, see [here](https://github.com/InFact-coop/YiMovi/blob/master/templates/views/director_profile.pug#L24)